common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -2094,6 +2084,16 @@ object CelebornConf extends Logging {
         "Allowed pattern is: `<host1>:<port1>[,<host2>:<port2>]*`")
       .createWithDefaultString(s"<localhost>:9097")
 
+  val MASTER_ENDPOINTS_RESOLVER: ConfigEntry[String] =
+    buildConf("celeborn.master.endpoints.resolver")
```

Comment:
Does `celeborn.master.endpoints.resolver` work for `celeborn.master.internal.endpoints`? If yes, please add more detail in doc.